src/Service/Resolvers/SqlMutationEngine.cs

```diff
@@ -528,6 +537,15 @@ private static OkObjectResult OkMutationResponse(JsonElement jsonResult)
                     (dbResultSet.Rows.FirstOrDefault() ?? new DbResultSetRow()) : null;
                 if (dbResultSetRow is not null && dbResultSetRow.Columns.Count == 0)
                 {
+                    if (operationType is Config.Operation.Create)
```

Comment:
What happens when you try an insert a record where the primary key already exists, and the request is allowed by DB policy? do you still fail with an authorization error?

Reply:
No, in that case we throw an exception, the particulars of which depend on the database flavour but is on the line of HttpStatusCode 409 (conflict) of primary key constraint.